Add stateless config paths.

Status: Not upstreamed, No upstream issue (not relevant at this time).

diff --git a/src/crypto/x509/root_linux.go b/src/crypto/x509/root_linux.go
index 8e79ccb..c7acb53 100644
--- a/src/crypto/x509/root_linux.go
+++ b/src/crypto/x509/root_linux.go
@@ -9,6 +9,7 @@ import "internal/goos"
 // Possible certificate files; stop after finding one.
 var certFiles = []string{
 	"/etc/ssl/certs/ca-certificates.crt",                // Debian/Ubuntu/Gentoo etc.
+	"/usr/share/defaults/etc/ssl/ca-certificates.crt",   // Serpent OS stateless
 	"/etc/pki/tls/certs/ca-bundle.crt",                  // Fedora/RHEL 6
 	"/etc/ssl/ca-bundle.pem",                            // OpenSUSE
 	"/etc/pki/tls/cacert.pem",                           // OpenELEC
@@ -18,8 +19,9 @@ var certFiles = []string{
 
 // Possible directories with certificate files; all will be read.
 var certDirectories = []string{
-	"/etc/ssl/certs",     // SLES10/SLES11, https://golang.org/issue/12139
-	"/etc/pki/tls/certs", // Fedora/RHEL
+	"/etc/ssl/certs",              // SLES10/SLES11, https://golang.org/issue/12139
+	"/usr/share/defaults/etc/ssl", // Serpent OS stateless
+	"/etc/pki/tls/certs",          // Fedora/RHEL
 }
 
 func init() {
